main.py

```diff
@@ -1,4 +1,14 @@
 import sys
+import os
+from crawler import *
+from dotenv import find_dotenv, load_dotenv
+
+dotenv_path = find_dotenv()
+load_dotenv(dotenv_path)
+
+
+def command_failed():
+    return None
```

Comment:
I would just exit() here, then replace all `return command_failed()` with just the call `command_failed()`

Reply:
[b0774e3]